runtime/doc/api.txt

```diff
@@ -9,8 +9,7 @@ Nvim API							   *API* *api*
 Nvim exposes a powerful API that can be used by plugins and external processes
 via |msgpack-rpc|, Lua and VimL (|eval-api|).
 
-Nvim can also be embedded in C applications as libnvim, so the application
-can control the embedded instance by calling the C API directly.
+Applications can also embed libnvim to control via the C ABI.
```

Comment:
I changed it to ABI to hint that any language with C bindings can use it. Why isn't it correct?